10_integrations/dbt_modal_inference/duck_db_modal_inference.py

```diff
@@ -0,0 +1,276 @@
+# ---
+# cmd: ["modal", "run", "10_integrations/dbt_modal_inference/duck_db_modal_inference.py"]
```

Comment:
this shouldn't be necessary, right? -- this should be the default command for anything testable